### PR TITLE
Test CI setup for kueue

### PR DESCRIPTION
**Pull request by ChughShilpa**

### CODE REVIEW

File: ci-operator/config/opendatahub-io/kueue/opendatahub-io-kueue-test.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    cluster_profile: aws-opendatahub
+    env:
+      BASE_DOMAIN: openshift-ci-aws.rhaiseng.com
+      HYPERSHIFT_AWS_REGION: us-west-2
+      HYPERSHIFT_NODE_COUNT: "3"
+    test:
+    - as: e2e-test-odh-kueue
+      cli: latest
+      commands: |
+        oc apply --server-side -k "github.com/opendatahub-io/kueue//config/default?ref=test"
```
Comment (astefanutti) [inline ci-operator/config/opendatahub-io/kueue/opendatahub-io-kueue-test.yaml:31]: Maybe we want to test the `rhoai` overlay, which is the one that's used in RHOAI.

Comment (ChughShilpa) [reply ci-operator/config/opendatahub-io/kueue/opendatahub-io-kueue-test.yaml:31]: I don't understand what do you mean by `rhoai overlay`. Could you please elaborate ?
@astefanutti

Comment (astefanutti) [reply ci-operator/config/opendatahub-io/kueue/opendatahub-io-kueue-test.yaml:31]: @ChughShilpa I mean `github.com/opendatahub-io/kueue/config/rhoai` instead.

Comment (ChughShilpa) [reply ci-operator/config/opendatahub-io/kueue/opendatahub-io-kueue-test.yaml:31]: Okay, understood now.

**Pull request closed without merging**